### ISSUE TITLE
Websocket closing a lot

### ISSUE THREAD

**Issue by ep1cman** [encounterplus/encounterplus#393]
I am trying to write an application that interacts with the Encounter+ websocket however I have hit an issue where the server seems to close the connection a lot. When I initially connect everything is fine and the connection remains open, however as soon as I do something in Encounter+ that would cause a message to be sent e.g. move a token, the web socket is closed. I have managed to hit this issue using both the python `websockets` library and the nodeJS `ws` library. Opening the web client and looking in the console, I see the web UI is also experiencing this issue.

<img width="541" alt="Screenshot 2021-03-06 at 15 29 19" src="https://user-images.githubusercontent.com/5303409/110211922-c6c32400-7e90-11eb-8913-15c549923145.png">

**Comment by ep1cman** [comment]
With some more testing, I think its related to having multiple connections open from the same source, e.g the web client and my custom script open on my laptop

**Comment by jurex** [comment]
it shouldn't matter @ep1cman. The only time when the socket should be disconnected is when your app goes to the background (on iPad/iPhone).

**Comment by ep1cman** [comment]
Hmm, it was happening pretty consistently when I opened the ticket (see the screenshot of logging output from the web client, but I also experienced it in my own python and NodeJS scripts). 

However today I am unable to reproduce the issue, so I'm happy to chalk it up to some MacOS weirdness happening that day. As soon as any message was supposed to be send from the server, all connections were suddenly cut off, but not with a proper web socket close if I had more than one open, but I just tried with 4 with no issues 🤷‍♂️

If I encounter it again I will capture more logs and re-open the issue.
